pkg/internal/snowflakeroles/snowflake_predefined_roles.go

```diff
@@ -3,6 +3,7 @@ package snowflakeroles
 import "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
 
 var (
-	Orgadmin     = sdk.NewAccountObjectIdentifier("ORGADMIN")
-	Accountadmin = sdk.NewAccountObjectIdentifier("ACCOUNTADMIN")
+	Orgadmin               = sdk.NewAccountObjectIdentifier("ORGADMIN")
+	Accountadmin           = sdk.NewAccountObjectIdentifier("ACCOUNTADMIN")
+	GenericScimProvisioner = sdk.NewAccountObjectIdentifier("GENERIC_SCIM_PROVISIONER")
```

Comment:
why not add also others (there were three in the docs, right?)

Reply:
hmm, I didn't need this atm, as the remaining roles are present in `defs` file. I can add them here in follow ups tho